internal/docker/docker_config.go

```diff
@@ -152,6 +152,16 @@ func CreateDockerCompose(s *types.Stack) *DockerComposeConfig {
 			Logging:       StandardLogOptions,
 		}
 		compose.Volumes[fmt.Sprintf("dataexchange_%s", member.ID)] = struct{}{}
+		if s.SandboxEnabled {
+			compose.Services["sandbox_"+member.ID] = &Service{
+				Image:         "ghcr.io/hyperledger/firefly-sandbox:latest",
```

Comment:
Noted that at some point we may need to adjust this after we decide the strategy around sandbox versioning with relation to FireFly itself